node/src/isolate/external.cpp

```diff
@@ -362,6 +397,22 @@ spawn_load_t::apply() {
 }
 
 void
+metrics_load_t::apply() {
+    auto _inner = inner;
+    stream.apply([&](decltype(stream.unsafe())& stream){
```

Comment:
It is taken by analogy from code of spool/spawn, mostly copy-pasted brainlessly, partly for code-base consistency. It seems that it'll not do any harm.